dotnet-authserver/src/TeacherIdentity.AuthServer/Infrastructure/ModelBinding/DateOnlyModelBinder.cs

```diff
@@ -30,11 +43,11 @@ public class DateOnlyModelBinderProvider : IModelBinderProvider
 {
     public IModelBinder? GetBinder(ModelBinderProviderContext context)
     {
-        if (context.Metadata.UnderlyingOrModelType == typeof(DateOnly) &&
-            (context.BindingInfo.BindingSource == BindingSource.Query ||
-            (context.BindingInfo.BindingSource == BindingSource.Path)))
+        if (context.Metadata.UnderlyingOrModelType == typeof(DateOnly))
```

Comment:
The removal of the check around `BindingSource` was to make the `[BindProperty(SupportsGet = true)] DateOnly DateOfBirth` property on the Details page work correctly with both query parameters and POST form submissions.